core/src/main/resources/reference.conf

```diff
@@ -28,11 +28,12 @@ akka.kafka.consumer = {
 }
 
 akka.kafka.committer = {
+    max-batch = 100000
     max-batch = ${?AKKA_KAFKA_COMMITTER_MAX_BATCH}
+    max-interval = 1 hour
     max-interval = ${?AKKA_KAFKA_COMMITTER_MAX_INTERVAL}
     parallelism = ${?AKKA_KAFKA_COMMITTER_PARALLELISM}
-    delivery = ${?AKKA_KAFKA_COMMITTER_DELIVERY}
```

Comment:
These configurations are being removed because they are hardcoded by the Kafka client itself so its impossible to override.